src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
Would this be better, for the sake of clarity and also in keeping with the intent of the original test:

```
assert.equal(element.localName, '<%= elementName %>');
```

That passes